pkg/apis/extensions/v1beta1/types.go

```diff
@@ -397,6 +397,10 @@ type DeploymentStatus struct {
 	// +optional
 	UpdatedReplicas int32 `json:"updatedReplicas,omitempty" protobuf:"varint,3,opt,name=updatedReplicas"`
 
+	// Total number of ready pods targeted by this deployment.
+	// +optional
+	ReadyReplicas int32 `json:"readyReplicas,omitempty"`
```

Comment:
Removing lgtm because this seems fishy - the protobuf tag is missing (note that everything is green in tests)
cc: @kubernetes/sig-api-machinery